Havit/Havit/Global/UIComponent/ContentsCollectionViewCell.swift

```diff
@@ -0,0 +1,167 @@
+//
+//  ContentsCollectionViewCell.swift
+//  Havit
+//
+//  Created by 박예빈 on 2022/01/12.
+//
+
+import UIKit
+import SnapKit
+import RxSwift
+
+class ContentsCollectionViewCell: UICollectionViewCell {
+    static var cellID = "ContentsCollectionViewCell"
+    
+    var mainImgView: UIImageView!
+    var alarmImgView: UIImageView!
+    var titleLabel: UILabel!
+    var subtitleLabel: UILabel!
+    var dateLabel: UILabel!
+    var linkLabel: UILabel!
+    var moreButton: UIButton!
+    var isReadImgView: UIImageView!
+    var alarmLabel: UILabel!
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+        setUpCell()
+        setAutoLayouts()
+    }
+    
+    override init(frame: CGRect) {
+        super.init(frame: .zero)
+        setUpCell()
+        setAutoLayouts()
+    }
+    
+    func setUpCell() {
+        mainImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        contentView.addSubview(mainImgView)
+        
+        titleLabel = {
+            let label = UILabel()
+            label.text = "슈슈슉 이것은 제목입니다 슈슉 슉슉 이것"
+            label.numberOfLines = 0
+            
+            return label
+        }()
+        contentView.addSubview(titleLabel)
+        
+        subtitleLabel = {
+            let label = UILabel()
+            label.text = "슈슈슉 이것은 제목입니다 슈슉 슉"
+            
+            return label
+        }()
+        contentView.addSubview(subtitleLabel)
+        
+        dateLabel = {
+            let label = UILabel()
+            label.text = "2021. 11. 24"
+            
+            return label
+        }()
+        contentView.addSubview(dateLabel)
+        
+        linkLabel = {
+            let label = UILabel()
+            label.text = "www.beansbin.oopy.io"
+            
+            return label
+        }()
+        contentView.addSubview(linkLabel)
+        
+        
+        alarmLabel = {
+            let label = UILabel()
+            label.text = "2021. 11. 17 오전 12:30 알림 예정"
+            
+            return label
+        }()
+        contentView.addSubview(alarmLabel)
+        
+        alarmImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        mainImgView.addSubview(alarmImgView)
+        
+        moreButton = {
+            let button = UIButton()
+            button.setTitle("...", for: .normal)
+            return button
+        }()
+        contentView.addSubview(moreButton)
+        
+        isReadImgView = {
+            let imgView = UIImageView()
+            imgView.image = UIImage()
+            
+            return imgView
+        }()
+        contentView.addSubview(isReadImgView)
+        
+    }
+    
+    func setAutoLayouts() {
+        mainImgView.snp.makeConstraints { (make) -> Void in
+            make.top.equalTo(contentView).offset(12)
```

Comment:
```suggestion
            make.top.equalToSuperview().inset(12)
```
P4;
contentView가 Cell 내부에 있는 View 맞죠? 
매번 contentView에 equal로 잡는 것보다 contentView와 superview가 같은 영역을 가지고 있으니 superview로 잡는건 어떨까요?

Reply:
contentView보다는 superView를 우선순위로 두라는 말씀이시군여 적용하겠습니다!